contracts/test/HardCodedConstantsTest.sol

```diff
@@ -0,0 +1,49 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.0;
+
+import { TokenManagerLiquidityPool } from '../token-manager/implementations/TokenManagerLiquidityPool.sol';
+import { Distributable } from '../utils/Distributable.sol';
+import { FlowLimit } from '../utils/FlowLimit.sol';
+import { NoReEntrancy } from '../utils/NoReEntrancy.sol';
+import { Operatable } from '../utils/Operatable.sol';
+import { Pausable } from '../utils/Pausable.sol';
+
+contract TestTokenManager is TokenManagerLiquidityPool {
+    constructor(address interchainTokenService_) TokenManagerLiquidityPool(interchainTokenService_) {
```

Comment:
add something other than constructor in each contract. otherwise deployed bytecode is the same as the original contract, confusing hardhat verifier.

Reply:
added a public constant variable `name` to each contract